crates/ruff/src/rules/flynt/helpers.rs

```diff
@@ -52,15 +49,20 @@ fn is_simple_callee(func: &Expr) -> bool {
 }
 
 /// Convert an expression to a f-string element (if it looks like a good idea).
-pub(super) fn to_f_string_element(expr: &Expr) -> Option<Expr> {
+pub(super) fn to_fstring_element(expr: &Expr) -> Option<ast::FStringPart> {
     match expr {
         // These are directly handled by `unparse_f_string_element`:
         Expr::Constant(ast::ExprConstant {
-            value: Constant::Str(_),
+            value: Constant::Str(value),
+            range,
             ..
-        })
-        | Expr::FString(_)
-        | Expr::FormattedValue(_) => Some(expr.clone()),
+        }) => Some(ast::FStringPart::String(ast::StringTodoName {
+            value: value.to_string(),
+            range: *range,
+        })),
+        // TODO (but "we don't know how to handle these right now" when called so not needed?)
+        // | Expr::FString(_)
```

Comment:
WHat part do we not know? How to handle FStrings?

Reply:
sorry for being unclear, this is a reference to an existing comment next to the only caller of this function, https://github.com/astral-sh/ruff/pull/6365/files/96513114ba006216c2601882692d9e96b96b5844#diff-18474f15dbc4220d441b5f36f7065939d7fa0acc0609b1b3a84d253db8211ae6L91-R92